ui/yarn.lock

```diff
@@ -5475,7 +5475,7 @@ axe-core@^4.10.0, axe-core@^4.2.0:
   resolved "https://registry.npmjs.org/axe-core/-/axe-core-4.10.0.tgz#d9e56ab0147278272739a000880196cdfe113b59"
   integrity sha512-Mr2ZakwQ7XUAjp7pAwQWRhhK8mQQ6JAaNWSjmjxil0R8BPioMtQsTLOolGYkji1rcL++3dCqZA3zWqpT+9Ew6g==
 
-axios@^1.6.1, axios@^1.7.7:
+axios@^1.6.1:
```

Comment:
so in the end we still have it in some sub dependencies :(

Reply:
I hope not in the production bundle 

```
yarn why v1.22.22
[1/4] 🤔  Why do we have the module "axios"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "axios@1.7.7"
info Reasons this module exists
   - "@storybook#test-runner#jest-playwright-preset#jest-process-manager#wait-on" depends on it
   - Hoisted from "@storybook#test-runner#jest-playwright-preset#jest-process-manager#wait-on#axios"
info Disk size without dependencies: "2.34MB"
info Disk size with unique dependencies: "2.48MB"
info Disk size with transitive dependencies: "2.87MB"
info Number of shared dependencies: 5
```

Reply:
`grep -r -n "axios" ui/dist/` is empty!